data/strings/strings.txt

```diff
@@ -1033,6 +1054,7 @@
     ja = すべての人のために無料で、愛を込めて作られています
     ko = 사랑으로 만든 모든 사람에게 무료
     lt = Nemokama visiems, sukurta su meile
+    lv = Brīva visiem, veidota ar mīlestību
```

Comment:
Does "free" here means that OM is: 
a) "costs nothing"
b) it's libre software
c) or that's available to everyone?

Reply:
All of these )

I would focus on the a) and c) meanings.

Reply:
"Bezmaksas un pieejama visiem, un veidota ar mīlestību"

Reply:
"Bezmaksas, pieejama visiem un veidota ar mīlestību"? :)